### PR TITLE
chore: lint and reformat the YAML files

### PR DESCRIPTION
**Pull request by behnazh**
Cleans up and reformats the YAML files using [pretty-format-yaml](https://github.com/macisamuele/language-formatters-pre-commit-hooks).

### CODE REVIEW

**Review by jenstroeger — APPROVED** [review]
Great stuff 🤓

**Comment by jenstroeger** [comment]
Hmm… should we call these files `.yml` or `.yaml`?

**Review by jenstroeger — APPROVED** [review]
Great stuff 🤓

One thought though… should we call the configuration files `.yml` or `.yaml`?

File: .github/workflows/check-and-release-main.yml [hunk 1]
```diff
@@ -4,14 +4,14 @@ name: Check and Release
 on:
   push:
     branches:
-      - master
+    - master
```
Comment (jenstroeger) [inline .github/workflows/check-and-release-main.yml:7]: See also PR https://github.com/jenstroeger/python-package-template/pull/72. Should we merge this PR first, and then rebase https://github.com/jenstroeger/python-package-template/pull/72 on top?

**Comment by behnazh** [comment]
> One thought though… should we call the configuration files `.yml` or `.yaml`?

I'm happy with `.yml`. Does it matter?

**Comment by jenstroeger** [comment]
Well… the YAML files use both extensions and I thought it would be nice to use one consistently. OCD 😳

File: .github/workflows/check-and-release-main.yml (hunk 1 above)
Comment (behnazh) [reply .github/workflows/check-and-release-main.yml:7]: Yes, sounds good.

**Comment by behnazh** [comment]
> Well… the YAML files use both extensions and I thought it would be nice to use one consistently. OCD flushed

Aren't we using `.yml` only in this repo?

**Comment by behnazh** [comment]
@jenstroeger I rebased this PR on `staging` but the `mypy` update fails as it used to [here](https://github.com/jenstroeger/python-package-template/actions/runs/1676449448). Do you have any idea what's going on?

**Comment by jenstroeger** [comment]
> Aren't we using `.yml` only in this repo?

Not for `.pre-commit-config.yaml` 😕

> I rebased this PR on `staging` but the `mypy` update fails as it used to [here](https://github.com/jenstroeger/python-package-template/actions/runs/1676449448). Do you have any idea what's going on?

Gah… `There are no .py[i] files in directory 'src'`? That’s very odd, let me rebase PR https://github.com/jenstroeger/python-package-template/pull/90 and see what happens…

File: .github/workflows/check-and-release-main.yml (hunk 1 above)
Comment (jenstroeger) [reply .github/workflows/check-and-release-main.yml:7]: Uhm?

**Comment by jenstroeger** [comment]
> If I rename `.pre-commit-config.yaml` to `.pre-commit-config.yml` I get […]

Oh Lord 🤦🏻‍♂️

I’d file a bug with [pre-commit](https://github.com/pre-commit/pre-commit) but then we both know that dude’s attitude 😉

**Comment by behnazh** [comment]
> > I rebased this PR on `staging` but the `mypy` update fails as it used to [here](https://github.com/jenstroeger/python-package-template/actions/runs/1676449448). Do you have any idea what's going on?
> 
> Gah… `There are no .py[i] files in directory 'src'`? That’s very odd, let me rebase PR #90 and see what happens…

The `mypy` issue should be fixed in #98.

**Pull request merged**